test/test_utils.py

```diff
@@ -492,11 +492,35 @@ class GetQtEnum(PicardTestCase):
 
     def test_get_qt_enum(self):
         from PyQt5.QtCore import QStandardPaths
-        values = util.get_qt_enum(QStandardPaths, QStandardPaths.LocateOption)
+        values = set(util.get_qt_enum(QStandardPaths, QStandardPaths.LocateOption))
         self.assertIn('LocateFile', values)
         self.assertIn('LocateDirectory', values)
         self.assertNotIn('DesktopLocation', values)
 
+    def test_get_qt_enum_no_match_or_not(self):
+        class SomeClass:
+            pass
+
+        class SomeAttrClass(list):
+            pass
+
+        class SomeOtherClass:
+            a = SomeAttrClass([1, 2, 3])
+            b = SomeAttrClass([4, 5, 6])
+
+        values = set(util.get_qt_enum(SomeClass, SomeAttrClass))
+        self.assertEqual(values, set())
+
+        values = set(util.get_qt_enum(SomeOtherClass, SomeAttrClass))
+        self.assertIn('a', values)
+        self.assertIn('b', values)
+
+        obj = SomeOtherClass()
+        res = []
+        for attr in sorted(util.get_qt_enum(SomeOtherClass, list)):
+            res.append(getattr(obj, attr))
+        self.assertEqual(res, [[1, 2, 3], [4, 5, 6]])
```

Comment:
I'd put the test for the three classes into three separate test functions. That makes it easier to handle failing tests.